app/src/main/java/com/randomappsinc/studentpicker/presentation/PresentationActivity.java

```diff
@@ -154,8 +164,12 @@ private void playSound() {
             UIUtils.showLongToast(R.string.drumroll_error, this);
         }
 
+        handleNamesAnimations(2600);
+    }
+
+    private void handleNamesAnimations(int delay) {
```

Comment:
should I change this to delayNamesAnimations?

Reply:
I think playNamesAnimation(int delayMs) is better. More descriptive variable and matches more of what the method does IMHO.